src/components/retrieve_from_weaviate/requirements.txt

```diff
@@ -0,0 +1,5 @@
+llama-index==0.8.68
+weaviate-client
+transformers
+torch
+fondant[docker]==0.6.2
```

Comment:
Can you use fondant 0.7? You will need `fondant[component]==0.7.0`

Reply:
Done, but it doesn't work when I set it in the load_from_csv component. I get an error saying that the main and fondant-component files cannot be found. It works when I put back to version 0.6.2. But version 0.7.0 works well for the retrieve component.
![image](https://github.com/ml6team/fondant-usecase-RAG/assets/117269156/401989fd-a2a2-4a18-a8fc-d68080e6d29c)